### PR TITLE
[streaming-movie-ratings] Updated to CP 5.5.1  and dbzm 1.2

### PR DESCRIPTION
**Pull request by gAmUssA**
updated docker dependencies to cp 5.5.1 and dbzm 1.2﻿

### CODE REVIEW

File: streams-movie-demo/legacy-movies-docker/docker-compose.yml [hunk 1]
```diff
@@ -152,45 +155,34 @@ services:
     command: "tail -f /dev/null"
 
   ksqldb-server:
-    image: confluentinc/ksqldb-server:0.6.0
+    image: confluentinc/ksqldb-server:0.9.0
```
Comment (rmoff) [inline streams-movie-demo/legacy-movies-docker/docker-compose.yml:158]: use 0.10?

File: streams-movie-demo/legacy-movies-docker/docker-compose.yml [hunk 2]
```diff
@@ -152,45 +155,34 @@ services:
...
-      KSQL_KSQL_STREAMS_PRODUCER_CONFLUENT_BATCH_EXPIRE_MS: 9223372036854775807
-      KSQL_KSQL_STREAMS_PRODUCER_REQUEST_TIMEOUT_MS: 300000
-      KSQL_KSQL_STREAMS_PRODUCER_MAX_BLOCK_MS: 9223372036854775807
-      KSQL_KSQL_STREAMS_REPLICATION_FACTOR: 3
-      KSQL_KSQL_INTERNAL_TOPIC_REPLICAS: 3
-      KSQL_KSQL_SINK_REPLICAS: 3
-
+      KSQL_KSQL_SCHEMA_REGISTRY_URL: "http://schema-registry:8081"
 
   ksqldb-cli:
```
Comment (rmoff) [inline streams-movie-demo/legacy-movies-docker/docker-compose.yml:175]: FWIW you don't need the CLI as a separate container, you can just invoke the CLI from the "server" image

File: streams-movie-demo/legacy-movies-docker/docker-compose.yml (hunk 1 above)
Comment (gAmUssA) [reply streams-movie-demo/legacy-movies-docker/docker-compose.yml:158]: didn't try 0.10. tho

File: streams-movie-demo/legacy-movies-docker/docker-compose.yml (hunk 2 above)
Comment (gAmUssA) [reply streams-movie-demo/legacy-movies-docker/docker-compose.yml:175]: really, this nice

File: streams-movie-demo/legacy-movies-docker/docker-compose.yml (hunk 1 above)
Comment (gAmUssA) [reply streams-movie-demo/legacy-movies-docker/docker-compose.yml:158]: TIL there are some breaking syntax changes in 0.10. I'll stick to 0.9 here :)

**Pull request merged**